scripts/pr_reviewer.py

```
@@ -1,104 +1,188 @@
 import os
 import re
-import requests
+import subprocess
+import tempfile
+import git
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)